src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/xresilience/StormLcmSpec.groovy

```diff
@@ -62,6 +62,20 @@ class StormLcmSpec extends HealthCheckSpecification {
         wfmManipulator = new WfmManipulator(dockerHost)
     }
 
+    def cleanupSpec() {
```

Comment:
added this topology readiness verification to avoid further tests execution when the topology has not been recovered after WFM restarting (we have dump verification that can fail before topology verification)